spec/workspace-spec.js

```diff
@@ -2086,7 +2086,7 @@ describe('Workspace', () => {
           })
 
           runs(() => {
-            fs.renameSync(path.join(projectPath, 'git.git'), path.join(projectPath, '.git'))
+            fs.rename(path.join(projectPath, 'git.git'), path.join(projectPath, '.git'))
```

Comment:
This change looks odd to me. Was `fs.rename` always synchronous in the old version of fs-extra?

Reply:
Prior to Node 10 `fs.rename` accepted a callback parameter and didn't complain if you didn't provide a callback. In Node 10 it seems that the callback is now mandatory if you use the non-Sync variant. I suppose the function worked as synchronous before if no callback was specified?